text/0033-view-hierarch.md

```diff
@@ -0,0 +1,56 @@
+* Start Date: 2022-11-02
+* RFC Type: feature
+* RFC PR: [#33](https://github.com/getsentry/rfcs/pull/33)
+* RFC Status: -
+* RFC Driver: [Dhiogo Ramos Brustolin](https://github.com/brustolin)
+
+# Summary
+
+Allow SDKs to send a textual representation of the application's UI view hierarch. 
+
+# Motivation
+
+Being able to see what the user was looking at is a good way to found out problems in the application. 
+This RFC proposes a way to send information from the UI state of an application, known as "view hierarch".
+
+# Proposal 
+
+The proposal is to create a new [envelope item](https://develop.sentry.dev/sdk/envelopes/) in the form of a json representation of the UI elements. 
+This is how the envelope item header should look like:
+
+```json
+{"type":"view_hierarch","length":...,"content_type":"application/json"}\n
```

Comment:
```suggestion
{"type":"view_ hierarchy","length":...,"content_type":"application/json"}\n
```

Reply:
Are you sure about this space?

Reply:
nope, GH suggestion messed up somehow, its about the `y` at the end, more places are missing.